.github/workflows/functional-tests.yml

```diff
@@ -63,10 +63,13 @@ jobs:
       - name: List cpu1
         run: ls build/exe/cpu1/
 
+      # Run cFS, send commands to set perf trigger and start perf data, and run functional tests
       - name: Run cFS
         run: |
           ./core-cpu1 &
           sleep 10
+          ../host/cmdUtil --pktid=0x1806 --cmdcode=17 --endian=LE --uint32=3 --uint32=0x40000000 &
+          ../host/cmdUtil --pktid=0x1806 --cmdcode=14 --endian=LE --uint32=2 &
```

Comment:
Remove `&` and bring commands to foreground to avoid race conditions

Reply:
Done